include/IECoreScene/private/PrimitiveAlgoUtils.h

```diff
@@ -48,8 +48,15 @@
 
 #include "boost/format.hpp"
 
+#include <tbb/version.h>
```

Comment:
The `tbb/version.h` header doesn't seem to exist in TBB 2020, so is breaking the build here :

```
src/IECore/IndexedIOAlgo.cpp:36:10: fatal error: tbb/version.h: No such file or directory
   36 | #include <tbb/version.h>
```